ethereum/contracts/BasicOutboundChannel.sol

```diff
@@ -56,9 +56,9 @@ contract BasicOutboundChannel is OutboundChannel, ChannelAccess, AccessControl {
      * Submission is a privileged action involving two parties: The operator and the origin (called account here).
      * Apps (aka operators) need to be authorized by the `account` to submit messages via this channel.
      */
-    function submit(address _account, bytes calldata _payload) external override {
-        require(isOperatorFor(msg.sender, _account), "Caller is unauthorized");
-        nonce[_account] = nonce[_account] + 1;
-        emit Message(msg.sender, _account, nonce[_account], _payload);
+    function submit(address account, bytes calldata payload, uint64) external override {
```

Comment:
Are we not using the `_variable` parameter naming convention anymore?

Reply:
yeah, not using the `_variable` convention anymore. It kills readability. I don't think the convention is useful in our case, because at an individual contract level, there are not many storage fields which can shadowed by parameters.